link/src/main/java/com/stripe/android/link/account/CookieStore.kt

```diff
@@ -84,18 +46,10 @@ class CookieStore @Inject internal constructor(
         }
 
     internal companion object {
-        const val AUTH_SESSION_COOKIE = "auth_session_cookie"
-        const val LOGGED_OUT_EMAIL_HASH = "logged_out_email_hash"
         const val SIGNED_UP_EMAIL = "signed_up_email"
 
         val allCookies = arrayOf(
-            AUTH_SESSION_COOKIE,
-            LOGGED_OUT_EMAIL_HASH,
             SIGNED_UP_EMAIL
         )
```

Comment:
Tiniest nit: Can get rid of this now.

Reply:
I wasn't sure if this was going to be the end state. But it's not a big change, so I'll just remove it for now.